pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -263,6 +264,7 @@ private void submitSegmentGenTask(File localTempDir, URI inputFileURI, int seqId
     taskSpec.setRecordReaderSpec(_spec.getRecordReaderSpec());
     taskSpec.setSchema(_schema);
     taskSpec.setTableConfig(_tableConfig);
+    ConsistentDataPushUtils.configureSegmentPostfix(_spec);
```

Comment:
Same as the push runners, we need to configure the postfix for all generation runners?

Reply:
Done.